plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.ts

```diff
@@ -64,6 +66,7 @@ export class OktaGroupEntityProvider extends OktaEntityProvider {
       parentGroupField?: string;
       namingStrategy?: GroupNamingStrategies | GroupNamingStrategy;
       userNamingStrategy?: UserNamingStrategies | UserNamingStrategy;
+      customAttributesToAnnotationWhitelist?: string[];
```

Comment:
Could you give an example of a profile field that you would not want to be loaded into backstage? It feels like the default should be to load them all in? Thoughts?

Reply:
i wanted to keep the previous behaviour as the default, and some organizations might not even need this, and adding everything as an annotation may generate too much clutter, i wanted to be minimalistic by default, we could have an option to just importa all as well